File: 2do Semestre/Aplica Estructuras de Control con un Lenguaje de Programacion/Programa C++/positivo-negativo.cpp
#include <stdio.h>
#include <stdlib.h>
#include <conio.h>
main ()
{
	int num;
	printf ("escribe un numero:");
	scanf ("%d",&num);
	if (num>0)
	printf ("es positivo");
	else
	printf ("es negativo");
	system ("pause");
	return 0;
}

File: 3er Semestre/Aplica la Metodologia Espiral con POO/Programas de NetBeans/positivonegativo/src/positivonegativo/Positivonegativo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package positivonegativo;
import java.util.Scanner;
/**
 *
 * @author PC-10
 */
public class Positivonegativo {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic her
    int num;
    Scanner teclado=new Scanner (System.in);
    System.out.println("Intoduce un numero:");
    num=teclado.nextInt();
    if (num>0){
       System.out.println("El numero es positivo");  
    }
    else {
       System.out.println("El numero es negativo");
    }
    }
    
}